target/linux/ipq806x/base-files/etc/init.d/bootcount

```diff
@@ -2,9 +2,9 @@
 
 START=99
 
-. "$IPKG_INSTROOT/lib/upgrade/asrock.sh"
```

Comment:
It should work just fine https://github.com/openwrt/openwrt/blob/master/include/rootfs.mk#L80-L87